scripts/python/SLiM_sims.py

```diff
@@ -6,32 +6,35 @@
 
 import subprocess
 import sys
+import os
 
 
-def parse(out):
-    lines = out.split("\n")
-    for line in lines:
-        print(line)
-        print("--------------")
+def run_slim(N, bot):
 
-
-def main():
-
-    N = int(sys.argv[1])  # initial N value
-    bottleneck = float(sys.argv[2])  # proportion of N
+    # Change directory to script path so it can be run from any directory in project
+    os.chdir(sys.path[0])
```

Comment:
Is this the accepted way to circumvent this directory issue? It seems a little unstable to me, but I may also very well be wrong

Reply:
Definitely not the accepted way. Was just a quick fix. I've removed this now.